File: C++/A1/046.cpp
#include <iostream>
#include <stdio.h>
using namespace std;
int main(){
    int N,A,sum=0,even=0,odd=0;
    cin >> N;
    for(int i=0;i<N;i++){
        cin  >> A;
        sum += A;
        if(A%2==0)even++;
        else odd++;
    }
    printf("SUM %d\nEVEN %d\nODD %d",sum,even,odd);
}

File: Python/A1/046.py
sum,even,odd=0,0,0
n = int(input())
lst = list(map(int,input().split()))
for i in range(n):
    sum += lst[i]
    if(lst[i]%2==0):even+=1
    else:odd+=1
print(f"SUM {sum}\nEVEN {even}\nODD {odd}")